docker-entrypoint.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
-[ ! -z ${LINK_PASSWORD} ] && echo "requirepass ${LINK_PASSWORD}" >> /tmp/redis.conf
+if [ ! -z "${LINK_PASSWORD}" ]; then
+  echo "requirepass ${LINK_PASSWORD}" >> /tmp/redis.conf
+else
+  echo "protected-mode no" >> /tmp/redis.conf
```

Comment:
Without this, you're not able to connect to connect to Redis from a non-loopback interface when there's no password set.